charts/camunda-platform/values.yaml

```diff
@@ -128,6 +128,9 @@ global:
 
       # Identity.auth.operate configuration to configure Operate authentication specifics on global level, which can be accessed by other sub-charts
       operate:
+        #Use another index-prefix for operate
+        elasticsearch:
```

Comment:
This is the wrong path.
It should be under `operate` not `global.identity.auth.operate`.

Reply:
@smartmanru this change is in the Helm chart value config, not in the app config. This is not the Operate configuration.
the values here are referenced in the Operate ConfigMap which has the actual Operate config.

Try the following in your branch to see it:
```
helm template charts/camunda-platform
```